test/out_dir_in_tests/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#[cfg(test)]
+mod tests {
+    use std::env;
+    use std::fs;
+    use std::path::PathBuf;
+
+    #[test]
+    fn can_find_the_out_dir_file() {
+        for (key, value) in env::vars() {
+            println!("{}: {}", key, value);
+        }
+        let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
```

Comment:
https://doc.rust-lang.org/cargo/reference/environment-variables.html says:
> OUT_DIR — If the package has a build script, this is set to the folder where the build script should place its output. See below for more information. (**Only set during compilation.**)

So I think this should be using `env!` instead of `env::var`.